docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/ConfigSection';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-The `expo-updates` library allows you to programmatically control and respond to new updates made available to your app.
+## Installation & Configuration
 
-## Installation
+`expo-updates` is a library that enables your app to manage remote updates to your application code. It communicates with the configured remote update service to get information about available updates.
+
+[EAS Update](/eas-update/introduction) is a hosted service that manages and serves updates, and includes tools to simplify installation and configuration of this library. To get started with EAS Update, follow the [Getting Started Guide](/eas-update/getting-started/).
+
+If you're installing this library in a [bare React Native app](/bare/overview/), follow these [additional installation instructions](/bare/installing-updates/).
+
```

Comment:
```suggestion
```

This should come inside the Manual instructions collapsible so we can avoid duplicating info (since this is also used in Manual installation collapsible below).